Makefile

```diff
@@ -1,12 +1,22 @@
-.PHONY: run setup deps-get db stop-db setup-wasm-prover
+.PHONY: run setup deps-get db stop-db setup-wasm-prover setup-assets start-db create-db
 
 run:
 	iex -S mix phx.server
 
-setup: deps-get db
+setup: submodule deps-get start-db setup-assets create-db
```

Comment:
@fkrause98 should this target (`setup`) also install wasm-pack (`cargo install wasm-pack`)?
I cloned this repo and ran `make setup` and it failed with `/bin/sh: wasm-pack: command not found`

Reply:
That's a good idea, I'll update the makefile